features/update/self/README.md

```diff
@@ -0,0 +1,10 @@
+# Workflow sends an update to itself
+
+Workflows can send updates back to themselves
+
+# Detailed spec
+
+A workflow can use the workflowservice client in an activity to send an update
```

Comment:
More a question on updates use case: In this test if there was some failure your activity to send a update could be called more then one correct? Could update validators be used to dedup the update if a UUID was attached?

Reply:
We made the choice not to include the UpdateID in the invocation of the handler but the user is certainly free to include an idempotency token in their application-layer update argument. The server can and does dedup by update ID in that a second request with the same update ID will wait on the same update protocol instance in the history service. You can think of this as finding and waiting on the same Future-ish Get()